### PR TITLE
Feature/add charging point frontend

### PR DESCRIPTION
**Pull request by SebastianZolkwer**
# :star: Feature: {Add charging point}

#### :link: https://github.com/SebastianZolkwer/obligatorio-agil2-SebastianZolkwer-MauroWynter-AlanGarfinkel/issues/63

---

#### :information_source: Description:

As an administrador you able to create a new charging point with description, address, identifier, name. region.

---

### CODE REVIEW

File: Obligatorio/MinTurFrontend/custom-e2e/specs/create-charging-point.feature [hunk 1]
```diff
@@ -0,0 +1,96 @@
+Feature: Test Alta Puntos de carga
+  Scenario: Creo un punto de carga con nombre largo
```
Comment (maurowynter) [inline Obligatorio/MinTurFrontend/custom-e2e/specs/create-charging-point.feature:2]: Que es nombre largo? Ser mas explicito

File: Obligatorio/MinTurFrontend/custom-e2e/specs/create-charging-point.feature [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  Scenario: Creo un punto de carga sin descripcion
+    Given I view the "http://localhost:4200/admin/chargingPoint/create"
+    When Ingreso el valor "punto de carga 1" en el campo "nombre"
+    When Ingreso el valor "Rambla gWhenhi 133" en el campo "direccion"
+    When Ingreso el valor "4785" en el campo "identificador"
+    When Selecciono el valor "Región Metropolitana" como región
+    When Registro el punto de carga
+    Then Veo el mensaje de error "La descripción debe tener un máximo de 60 caracteres"
+
+  Scenario: Creo un punto de carga con descripcion larga
```
Comment (maurowynter) [inline Obligatorio/MinTurFrontend/custom-e2e/specs/create-charging-point.feature:30]: Descripcion "larga", idem anterior comentario

File: Obligatorio/MinTurFrontend/custom-e2e/specs/create-charging-point.feature [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    Given I view the "http://localhost:4200/admin/chargingPoint/create"
+    When Ingreso el valor "punto de carga 1" en el campo "nombre"
+    When Ingreso el valor "Rambla gWhenhi 133" en el campo "direccion"
+    When Ingreso el valor " El punto de carga mejor de toda la ciudad, evndemos electricidad para los autos para que se puedan recargar facilmente" en el campo "descripcion"
+    When Ingreso el valor "4785" en el campo "identificador"
+    When Selecciono el valor "Región Metropolitana" como región
+    When Registro el punto de carga
+    Then Veo el mensaje de error "La descripción debe tener un máximo de 60 caracteres"
+
+  Scenario: Creo un punto de carga con direccion larga
```
Comment (maurowynter) [inline Obligatorio/MinTurFrontend/custom-e2e/specs/create-charging-point.feature:40]: Idem

File: Obligatorio/MinTurFrontend/src/app/app-routing.module.ts [hunk 4]
```diff
@@ -36,9 +36,13 @@ import { ReservationReportComponent } from './feature-modules/reservation-report
 import { ReservationReportModule } from './feature-modules/reservation-report/reservation-report.module';
 import { ImportingComponent } from './feature-modules/importing/importing.component';
 import { ImportingModule } from './feature-modules/importing/importing.module';
+import { ChargingPointListComponent } from './feature-modules/charging-point-list/chargingPoint-list.component';
+import { ChargingPointCreateComponent } from './feature-modules/charging-point-create/charging-point-create.component';
+import { ChargingPointCreateModule } from './feature-modules/charging-point-create/charging-point-create.module';
 // tslint:disable-next-line:max-line-length
```
Comment (maurowynter) [inline Obligatorio/MinTurFrontend/src/app/app-routing.module.ts:42]: Para que quedo este comentario?

Comment (SebastianZolkwer) [reply Obligatorio/MinTurFrontend/src/app/app-routing.module.ts:42]: Porque sino falla eslint

**Review by alangarfinkel — APPROVED** [review]

**Pull request merged**